packages/@aws-cdk/aws-scheduler-alpha/test/integ.schedule.ts

```diff
@@ -1,21 +1,24 @@
 /// !cdk-integ aws-cdk-scheduler-schedule
-import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { ExpectedResult, IntegTest, Match } from '@aws-cdk/integ-tests-alpha';
 import * as cdk from 'aws-cdk-lib';
 import * as cloudwatch from 'aws-cdk-lib/aws-cloudwatch';
 import * as iam from 'aws-cdk-lib/aws-iam';
 import * as kms from 'aws-cdk-lib/aws-kms';
-import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
 import * as scheduler from '../lib';
 
-class SomeLambdaTarget implements scheduler.IScheduleTarget {
-  public constructor(private readonly fn: lambda.IFunction, private readonly role: iam.IRole) {
+class SomeSqsTarget implements scheduler.IScheduleTarget {
```

Comment:
Opted to use SQS queue for this test so that we can use `ReceiveMessage` in the assertions to test the runtime behaviour, without needing to write a Lambda Function.